clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/FieldActions.const.tsx

```diff
@@ -135,8 +178,8 @@ export const FIELD_ACTION_HOTKEYS = {
 export const FIELD_ACTION_CONFIRMATION_MESSAGE = {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   "assign-categories": (_targetItemCount: number) => null,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  "promote-removals": (_targetItemCount: number) => null,
+  "promote-removals": (targetItemCount: number) =>
+    `Are you sure you want to remove these ${targetItemCount.toLocaleString()} resources?`,
```

Comment:
This will end up being weird if only 1 item is selected
```suggestion
  "promote-removals": (targetItemCount: number) =>
    `Are you sure you want to remove ${targetItemCount.toLocaleString()} ${pluralize(targetItemCount, "resource", "resources")}?`,
```